docs/tuning-for-production.md

```diff
@@ -6,6 +6,20 @@ title: Tuning the ProxyManager for production
 
 By default, all proxy factories generate the required proxy classes at runtime.
 
+To generate proxies and store them as files, you need to use `FileWriterGeneratorStrategy` by passing it to the configuration before setting up other factories:
```

Comment:
/to use/to use the

Reply:
`the configuration` is unclear. Maybe just saying `configuring ProxyManager`?

Reply:
Not sure what your second comment means. Could you clarify?

Reply:
`the configuration` refers to something specific, yet you don't name it here. I suggest removing `by passing it to the configuration before setting up other factories:` and having `by configuring ProxyManager:`